styles.css

```diff
@@ -0,0 +1,11 @@
+rect {
```

Comment:
I think this is a stray — this file should be in the styles/ directory (Farhan's change moved it there).